app/controllers/api/v2/compute_resources_controller.rb

```diff
@@ -144,7 +144,7 @@ def available_zones
       param :id, :identifier, :required => true
       param :cluster_id, String
       def available_networks
-        @available_networks = @compute_resource.available_networks(params[:cluster_id].presence)
+        @available_networks = @compute_resource.available_networks(cluster_id)
```

Comment:
The cluster_id is defined by apipie?

Reply:
We have cluster_id in the controller, here is where we are using it to pass to the argument you listed:

https://github.com/theforeman/foreman/pull/9383/files#diff-4c9e5d9690c2678a2aa9086871b8b0057f0dcbd191112127dfb2b7379020d0dfR298-R300

I just moved it is all so we could escape it

Reply:
Oh! right, totaly it makes total sense then to do this! :) I did not see that method definition.